install/install.md

```diff
@@ -162,47 +225,48 @@ bash /rapids/utils/stop-jupyter.sh
 
 
 <br/>
+<div id="pip"></div>
 
-### **pip**
-The package installer for python (pip) is available as of RAPIDS release 23.04. See below for details.
-
+## **Pip**
+Beginning with the release of 23.04: cuDF, dask-cuDF, cuML, cuGraph, RMM, and RAFT CUDA 11 pip packages are available on the NVIDIA Index.
 
-<br/>
-<div id="source"></div>
+### **Pip Enhanced Prerequisites**
```

Comment:
What is an "enhanced prerequisite"? This sounds like an optional feature but it is not optional. Perhaps "Additional requirements for pip packages"? Also applies to WSL2 section.

Reply:
Agree 'Enhanced' isn't immediately clear.

I like `pip - additional requirements` it's short, and starts with pip so it's easier to find when skimming